source/app_setup.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+pushd ../
+conda env update --file local-environment.yml;
```

Comment:
This only works if the user has conda env installed and the memex environment activated.  You need:

(install latest version of conda if it's not installed yet)
then:  `conda env create --name memex --file /vagrant/environment.yml`